src/main/java/com/google/cloud/discotoproto3converter/proto3/DocumentToProtoConverter.java

```diff
@@ -378,6 +378,12 @@ private void readResources(Document document) {
     }
   }
 
+  private Option getFieldBehaviorOption() {
```

Comment:
Please rename to something like `createFieldBehaviorOptionRequired` to clarify that it _creates_ an option, and specifically, the `REQUIRED` option.

Reply:
Made the value of the option a parameter